hydrus/item_collection_helpers.py

```diff
@@ -14,7 +14,10 @@
     InstanceNotFound,
     PageNotFound,
     InvalidSearchParameter,
-    OffsetOutOfRange)
+    OffsetOutOfRange,
+    DatabaseConstraintError,
+    PropertyNotGiven
+)
```

Comment:
See #481 for line 89

Reply:
Yes, I understood the problem we have atm with notation of Collection Resources. I will make a new PR to fix this issue